cluster/cluster_test.go

```diff
@@ -7,46 +7,56 @@ import (
 )
 
 func TestPeersForQuery(t *testing.T) {
-	Init("node1", "test", time.Now())
-	ThisNode.SetPrimary(true)
-	ThisNode.SetPartitions([]int32{1, 2})
-	ThisNode.SetReady()
+	Init("node1", "test", time.Now(), "http", 6060)
+	Manager.SetPrimary(true)
+	Manager.SetPartitions([]int32{1, 2})
+	Manager.SetReady()
 	Convey("when cluster in single mode", t, func() {
 		selected := PeersForQuery()
 		So(selected, ShouldHaveLength, 1)
-		So(selected[0], ShouldEqual, ThisNode)
+		So(selected[0], ShouldResemble, Manager.ThisNode())
```

Comment:
[ShouldResemble](https://godoc.org/github.com/smartystreets/assertions#ShouldResemble) is just a deep equal as you cant do equality checks on structs that contain pointers (or maps or slices).   You can do equality on references though, as you are just comparing the memory addresses.